### PR TITLE
fix: standardize upgrade cordon-drain timeout

### PR DESCRIPTION
**Pull request by jackfrancis**
**Reason for Change**:

Enable future maintenance of cordon/drain timeout by making it a discrete const.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/operations/kubernetesupgrade/upgradeagentnode.go [hunk 1]
```diff
@@ -19,8 +19,9 @@ import (
 )
 
 const (
-	interval = time.Second * 1
-	retry    = time.Second * 5
+	interval           = time.Second * 1
+	retry              = time.Second * 5
+	cordonDrainTimeout = time.Minute * 10
```
Comment (CecileRobertMichon) [inline pkg/operations/kubernetesupgrade/upgradeagentnode.go]: why 10 minutes? if scale is 1 hour

Comment (jackfrancis) [reply pkg/operations/kubernetesupgrade/upgradeagentnode.go]: This is just a placeholder to demonstrate the surface area that needs changing. No concrete requirement yet.

**Comment by CecileRobertMichon** [comment]
any blockers for moving this forward?

**Comment by jackfrancis** [comment]
@CecileRobertMichon I reverted the functional change from 1 minute to 10 minutes. Now the scope of this PR is just to move the timeout value to be more maintainable as a discrete const.

**Comment by jackfrancis** [comment]
I think we want to have definitive repro cases where increasing the timeout causes a demonstrable improvement (with no tradeoffs).

**Pull request closed without merging**